packages/flutter_tools/lib/src/web/devfs_web.dart

```diff
@@ -79,11 +80,17 @@ class WebAssetServer {
       await response.close();
       return;
     }
+    // TODO(jonahwilliams): better path normalization in frontend_server to remove
+    // this workaround.
+    String requestPath = request.uri.path;
+    if (requestPath.startsWith('/C:/')) {
```

Comment:
So, it won't work for '/D:/'? At home my C drive is full, so I mostly use D and E :smiley:

Reply:
Ahh, RegExp my old friend....